common/app/templates/inlineJS/nonBlocking/emailIframeTracking.scala.js

```diff
@@ -16,15 +18,52 @@ const sendEvent = (payload, eventType) => {
 
 const getClickEvent = (el) => {
     return {
-        clickComponent: el.getAttribute('data-component'), clickLinkNames: [el.getAttribute('data-link-name')]
+        clickComponent: el.getAttribute('data-component'),
+        clickLinkNames: [el.getAttribute('data-link-name')]
     }
 }
 
+const buildComponentEvent = (formElement, actionType, actionDescription) => ({
+    componentEvent: {
+        component: {
+            componentType: 'NEWSLETTER_SUBSCRIPTION',
+            id: formElement.getAttribute('data-component'),
+        },
+        action: actionType,
+        value: [actionDescription,formElement.getAttribute('data-email-list-name')].join(),
+    }
+});
 
-const trackClickEvent = (el) => {
-    el.addEventListener('click', (event) => {
-        const clickEvent = getClickEvent(el)
-        sendEvent(clickEvent, 'click-event')
+const trackClickEvent = (buttonElement) => {
+    if (!buttonElement) { return {} }
+    buttonElement.addEventListener('click', (event) => {
+        const clickEvent = getClickEvent(buttonElement)
+        sendEvent(clickEvent, 'ophan-iframe-click-event')
     })
 }
 
+function validateForm() {
```

Comment:
Just wondering why we're using the `function` keyword here instead of the `=>` operator for declaring functions, or viceversa, worth making it consistent?

```js
function asdf() {}
```
vs
```js
const asdf = () => {}
```

Reply:
Thanks for the review! There was no reason for this, it was just that I moved some code from a couple of places into this new script and there was a difference in style - but I agree, it's best to stick to one or the other. I think `function` declarations are a better fit, so I've change to use those. 
I've also renamed some of the functions and variables as the the names weren't too clear.